test/Cli/Func.E2ETests/Commands/FuncStart/TestsWithFixtures/DotnetIsolatedTests.cs

```diff
@@ -0,0 +1,145 @@
+
+            // Validate that getting http endpoint works
+            capturedContent.Should().Be(
+                "Welcome to Azure Functions!",
+                because: "response from default function should be 'Welcome to Azure Functions!'");
+
+            // Validate default host was started
+            result.Should().HaveStdOutContaining("4.10");
+            result.Should().HaveStdOutContaining("Selected default host.");
+        }
+
+        [Fact]
+        [Trait(TestTraits.Group, TestTraits.UseInConsolidatedArtifactGeneration)]
+        public void Start_DotnetIsolated_WithoutRuntimeSpecified()
+        {
+            int port = ProcessHelper.GetAvailablePort();
+
+            // Call func start
+            var funcStartCommand = new FuncStartCommand(_fixture.FuncPath, "Start_DotnetIsolated_WithoutRuntimeSpecified", _fixture.Log);
+            string? capturedContent = null;
+
+            funcStartCommand.ProcessStartedHandler = async (process) =>
+            {
+                capturedContent = await ProcessHelper.ProcessStartedHandlerHelper(port, process, funcStartCommand.FileWriter, "HttpTrigger?name=Test");
+            };
+
+            var result = funcStartCommand
+                        .WithWorkingDirectory(_fixture.WorkingDirectory)
+                        .WithEnvironmentVariable("FUNCTIONS_WORKER_RUNTIME", "dotnet-isolated")
+                        .Execute(new[] { "--verbose", "--port", port.ToString() });
+
+            // Validate that getting http endpoint works
+            capturedContent.Should().Be(
+                "Welcome to Azure Functions!",
+                because: "response from default function should be 'Welcome to Azure Functions!'");
+
+            // Validate default host was started
+            result.Should().HaveStdOutContaining("4.10");
+            result.Should().HaveStdOutContaining("Selected out-of-process host.");
+        }
+
+        [Fact]
+        public void DontStart_InProc6_SpecifiedRuntime_ForDotnetIsolated()
+        {
+            int port = ProcessHelper.GetAvailablePort();
+
+            // Call func start with invalid runtime (expected to fail)
+            var result = new FuncStartCommand(_fixture.FuncPath, "DontStart_InProc6_SpecifiedRuntime_ForDotnetIsolated", _fixture.Log)
+                        .WithWorkingDirectory(_fixture.WorkingDirectory)
+                        .WithEnvironmentVariable("FUNCTIONS_WORKER_RUNTIME", "dotnet-isolated")
+                        .Execute(new[] { "--verbose", "--runtime", "inproc6", "--port", port.ToString() });
+
+            // Validate error message
+            result.Should().ExitWith(1);
+            result.Should().HaveStdErrContaining("The runtime argument value provided, 'inproc6', is invalid. The provided value is only valid for the worker runtime 'dotnet'.");
+        }
+
+        [Fact]
+        public void DontStart_InProc8_SpecifiedRuntime_ForDotnetIsolated()
```

Comment:
We are still using the func start command no? 

So following the pattern I mentioned eariler, this would be:

`Start_DotnetIsolatedApp_With_InProc8AsRuntime_ShouldFail`